### ISSUE TITLE
Improve events navigation / details

### ISSUE THREAD

**Issue by olensmar** [kubeshop/monokle#3657]
selecting an Event from the Overview currently just takes me to the list of events - now I have to manually try to find the event that I clicked on to see its details. We need to improve this workflow:
- the selected event should be highlighted and "expanded" - so I can directly dig into its details
- the list of events should contain some more info / columns:
  - count
  - object (with a link to the involved object)
  - reason ? 
- the Info tab of the event should also contain a direct link to the involved-object - together with the message and reason perhaps!?

**Comment by olensmar** [comment]
Perhaps a dedicated "Events" panel is the way to go - with filtering on reason, involved-object, etc!?

**Comment by chargio** [comment]
I agree that the dedicated event panel  (and the event panel in the objects), is the way to go.
Events are time based construct. you want to know the history of what has happened (like logs), in addition to the current state (if there is some alarm firing). 

Currently, as there can be many events, it is hard to use them and they behave like noise. A dedicated panel will allow the user to search and understand what is happening. A dedicated panel for events in a pod let me know if there is some error (i..e the pod has been rebooting every 5 seconds for the last 30 minutes).

**Comment by chargio** [comment]
Linked to #3637
